### ISSUE TITLE
Backup scheduling not triggering on new install

### ISSUE THREAD

**Issue by mericanpi** [thunderbird/import-export-tools-ng#564]
Ubuntu 20.04.6 LTS
Thunderbird 115.10.1 (64-bit)
Importexporttoolsng v14.0.2

I have just set up this computer and hoped that it would create an automated backup of our emails each day. I have set up scheduling, but the only way I can get the emails to back up is to run a manual backup. When I do it all functions just fine. 

Are there any things I should check? I didn't see anyone else with this issue so I hoped that I could find an answer here. 

Not sure how to get log, if anyone knows the command to do that I'm happy to oblige. Here is screenshot of my settings. 
![Screenshot from 2024-05-06 00-12-00](https://github.com/thunderbird/import-export-tools-ng/assets/33487487/ba8ba4b2-a978-42e4-b456-8628d11c189d)

**Comment by Alexey71** [comment]
```
Start backup check
WebExtensions: OverlayBackup [messengerOL.js:90](chrome://mboximport/content/mboximport/messengerOL.js)
WebExtensions: 135206.63100004196 [messengerOL.js:91](chrome://mboximport/content/mboximport/messengerOL.js)
WebExtensions: 86400 [messengerOL.js:92](chrome://mboximport/content/mboximport/messengerOL.js)
Error: sendRemoveListener on closed conduit ImportExportToolsNG@cleidigh.kokkini.net.137438953474 [ConduitsChild.sys.mjs:108:13](resource://gre/modules/ConduitsChild.sys.mjs)
    _send resource://gre/modules/ConduitsChild.sys.mjs:108
    removeListener resource://gre/modules/ExtensionChild.sys.mjs:675
    removeListener resource://gre/modules/ExtensionChild.sys.mjs:940
    unsetListener chrome://messenger/content/child/ext-menus.js:80
    close chrome://messenger/content/child/ext-menus.js:113
    unload resource://gre/modules/ExtensionCommon.sys.mjs:979
    unload resource://gre/modules/ExtensionPageChild.sys.mjs:282
    unload resource://gre/modules/ExtensionPageChild.sys.mjs:319
    destroyExtensionContext resource://gre/modules/ExtensionPageChild.sys.mjs:479
    observe resource://gre/modules/ExtensionPageChild.sys.mjs:388
```

**Comment by cleidigh** [comment]
@mericanpi 
The debug looks ok.
I have a beta for v14.0.3 that has some more debug for backup on shutdown.
There is some other work in progress so if you run into issues, downgrade.
Try capturing the output on disable again.
https://github.com/thunderbird/import-export-tools-ng/blob/v14.0.3/xpi/beta/import-export-tools-ng-14.0.3-b4-tb.xpi

@cleidigh

**Comment by mericanpi** [comment]
[console-export-2024-5-9_8-32-27.txt](https://github.com/thunderbird/import-export-tools-ng/files/15255647/console-export-2024-5-9_8-32-27.txt)
I apologize, I was AFK for a couple of days. Here is the output.

**Comment by mericanpi** [comment]
Yes, backup on shutdown is working, but I really need it to run automatically and passively without human interference.

**Comment by cleidigh** [comment]
@mericanpi 
Ok, maybe I misunderstood. I thought manual backup with the button was working, but auto backup on shutdown did not. Is that correct?
Are you asking about backup on schedule without a shutdown?
If so, the backup does not currently support a backup without the shutdown.
@cleidigh

**Comment by mericanpi** [comment]
Thank you. Sorry for slow responses. Sometimes can be called away for a few days - hence I need automated backups.

Is there a cron job that may be able to trigger them? Just wondering. No worries if not. When you get it enabled without shutdown that would be a great feature.
